crates/cheatcodes/src/inspector.rs

```diff
@@ -523,12 +525,18 @@ impl Cheatcodes {
             ignored_traces: Default::default(),
             arbitrary_storage: Default::default(),
             deprecated: Default::default(),
+            wallets: Default::default(),
         }
     }
 
     /// Returns the configured script wallets.
-    pub fn script_wallets(&self) -> Option<&ScriptWallets> {
-        self.config.script_wallets.as_ref()
+    pub fn script_wallets(&self) -> Option<&Wallets> {
```

Comment:
nit, maybe instead `script_wallets` and `set_wallets` fns we could have single
```rust
pub fn script_wallets(&mut self) -> &Wallets {
    self.wallets.get_or_insert(Wallets::new(MultiWallet::default(), None))
}
```

then we can simplify `inject_wallet` to
```rust
fn inject_wallet(state: &mut Cheatcodes, wallet: LocalSigner<SigningKey>) -> Address {
    let address = wallet.address();
    state.script_wallets().add_local_signer(wallet);
    address
}
```

and `getWalletsCall` to

```rust
impl Cheatcode for getWalletsCall {
    fn apply_stateful(&self, ccx: &mut CheatsCtxt) -> Result {
        let signers = ccx.state.script_wallets().signers().unwrap_or_default();
        Ok(signers.abi_encode())
    }
}
```

wdyt?

Reply:
oops didn't notice the auto-merge, @yash-atreya please take a look, I think this is reasonable

Reply:
Sorry @grandizzy, it was on auto-merge. I'll make a follow up PR